src/Components/AffectedClustersTable/AffectedClustersTable.spec.ct.js

```diff
@@ -7,6 +7,7 @@ import { AffectedClustersTable } from './AffectedClustersTable';
 import props from '../../../cypress/fixtures/AffectedClustersTable/data.json';
 import { Intl } from '../../Utilities/intlHelper';
 import getStore from '../../Store';
+import { array } from 'prop-types';
```

Comment:
It seems like imported by accident. Please, remove it since it is not used anywhere

Reply:
This is actually the cause for the lint failure in your build :-D

Reply:
Good catch! Thank you